app/views/surveys/_form.html.erb

```diff
@@ -0,0 +1,75 @@
+<%= fields_for :survey, survey do |f|  %>
+  <div class="row">
+    <div class="col">
+      <div class="form-group">
+        <label for="description">Titel (deutsch)</label>
+        <%= f.text_field :title_de, name: "survey[title][de]", class: "form-control" %>
```

Comment:
Was hat es mit den deutsch-polnischen Inputs auf sich?

Reply:
Gute Frage. Das Umfrage-Feature ist explizit für Deutsch und Polnisch beauftragt worden. Wir haben somit die Möglichkeit von Umfrage-Feldern für diese zwei Sprachen geschaffen. Das zieht sich durch ein paar Elemente bis hin zur Übertragung der Daten per API an den Server.

Reply:
in einem späteren commit von marco werden die sprachen etwas dynamisiert: https://github.com/ikuseiGmbH/smart-village-app-cms/pull/76/commits/9a251e588d019e9817f07208cffa72469c058954